backend/api/Areas/Keycloak/Controllers/UserController.cs

```diff
@@ -107,6 +107,20 @@ public async Task<IActionResult> UpdateUserAsync(Guid id, [FromBody] Model.Updat
 
             return new JsonResult(result);
         }
+
+        /// <summary>
+        /// Update an access request, generally to grant/deny it.
+        /// </summary>
+        /// <returns></returns>
+        [HttpPut("access/request")]
+        public async Task<IActionResult> UpdateAccessRequests(Pims.Api.Models.AccessRequestModel updateModel)
```

Comment:
Rename to `UpdateAccessRequestsAsync` to keep in alignment with other async naming pattern.